reconcile/change_owners/decision.py

```diff
@@ -106,28 +141,37 @@ def apply_decisions_to_changes(
     to generate the coverage report and to reason about the approval
     state of the MR.
     """
+    approvers_decisions_by_name = {
+        d.approver_name: d.command for d in approver_decisions
+    }
+
     diff_decisions = []
     for c in changes:
         for d in c.diff_coverage:
             change_decision = ChangeDecision(
                 file=c.fileref, diff=d.diff, coverage=d.coverage
             )
             diff_decisions.append(change_decision)
+
             for change_type_context in change_decision.coverage:
                 # approvers of a disabled change-type are ignored
                 if change_type_context.disabled:
                     continue
 
-                # autoapproval authors
-                if (
+                # a context is auto-approved if
+                #  * it has only one approver
```

Comment:
I don't understand this condition:
> it has only one approver
Why can't we have other possible approvers for (part of) the MR?